pkgs/development/python-modules/pygrok/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, regex }:
+
+buildPythonPackage rec {
+  pname = "pygrok";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1bqhb1pyd2c9rgbc4pjhf673x23hmcdsx1lxximbgsm01cy5wqxf";
+  };
+
+  propagatedBuildInputs = [ regex ];
+
```

Comment:
please add tests, they are important to make sure you aren't broken by dependency bumps. May need to checkout from github

```
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
Finished executing setuptoolsCheckPhase
/nix/store/bjqg9szrr90xip7g88habsc1vi2i08lz-python3.7-pygrok-1.0.0
```

Reply:
@jonringer Thank you for your feedback. 

I switched to the GitHub release tarball and have overwritten the checkPhase to run the test suite.

Reply:
they may be running, but i dont see any output, do you mind changing this to:

```
  checkInputs =  [ pytest ];
  checkPhase = ''
    pytest
  '';
```

Reply:
Good catch. I switched to pytest.